migrations/Version202608041200001488_taoAdvancedSearch.php

```
@@ -0,0 +1,53 @@
+            }
+        } catch (Throwable $exception) {
+            $this->addReport(
+                Report::createError(
+                    sprintf('Failed adjusting item comments to Elasticsearch: %s', $exception->getMessage())
+                )
+            );
+            throw $exception;
+        }
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)